src/Tribe/Events/Views/V2/Models/Tickets.php

```diff
@@ -109,7 +110,16 @@ public function fetch_data() {
 			return $this->data;
 		}
 
-		if ( ! tribe_events_has_tickets_on_sale( $this->post_id ) ) {
+		$num_ticket_types_available = 0;
+		foreach( $this->all_tickets as $ticket ) {
+			if ( ! tribe_events_ticket_is_on_sale( $ticket ) ) {
+				continue;
+			}
```

Comment:
* :no_entry_sign: There must be a space between foreach and the first parenthesis '('
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after FOREACH keyword; 0 found

<sub>hash: e2db76660c0d00532277816ffebcb992</sub>